src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -318,6 +324,16 @@ internal override void SetDefaultSourcePath(string defaultPath)
             WriteValueToFile<string>(fileName, "DefaultSourcePath", defaultPath);
         }
 
+        /// <summary>
+        /// Allow loading assemblies by searching in GAC.
+        /// </summary>
+        /// <returns>Boolean indicating whether assemblies should be loaded from the GAC</returns>
+        internal override bool GetAllowGacLoading()
+        {
+            string fileName = Path.Combine(psHomeConfigDirectory, configFileName);
+            return ReadValueFromFile<bool>(fileName, "AllowGacLoading");
```

Comment:
@SteveL-MSFT @joeyaiello I believe it was discussed to be opt-in due to performance implications. From what I see, the performance difference is not noticeable. So should I change it to opt-out?